db/migrate/20220202141625_create_users.rb

```diff
@@ -1,7 +1,7 @@
 class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
-      t.string :name, null: false
+      t.string :name, null: false 
```

Comment:
Ficou um espaço a mais aqui

```suggestion
      t.string :name, null: false
```